exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/ManagedChannelTelemetryExporterBuilder.java

```diff
@@ -158,16 +160,19 @@ public TelemetryExporterBuilder<T> setChannel(Object channel) {
 
   @Override
   public TelemetryExporter<T> build() {
-    requireNonNull(channelBuilder, "channel");
+    AtomicReference<Runnable> shutdownCallbackRef = new AtomicReference<>(() -> {});
```

Comment:
took me a bit to realize the AtomicReference here was just used to make an effectively final var to use in anonymous class below

it might be clearer

```suggestion
    Runnable shutdownCallback;
    if () {
      ...
      shutdownCallback = channel::shutdownNow;
    } else {
      shutdownCallback = () -> {};
    }
```